modules/doc/content/framework_development/code_standards.md

```diff
@@ -319,4 +319,4 @@ class MyClass:
 - All function definitions should be in *.C files.
     - The only exceptions are for inline functions for speed and templates.
 - Thou shall not commit accidental insertion in a std::map by using brackets in a RHS operator unless he/she can prove it can't fail.
-- Thou shall use range-based loops or MooseIndex() based loops for iteration
\ No newline at end of file
+- Thou shall use range-based loops or MooseIndex() based loops for iteration
```

Comment:
Maybe outside the scope of this PR but this bizarre mix of active and passive voice, and old and new English is cringe-worthy. Just about anything sounds better to me than `Thou shall`...like `Thou shalt` or `You shall` or we could use passive voice like up above. Anyway we should do that in a different PR (I have to be honest...I don't even see where the diff is here)

Reply:
My vote is for "Þou shalt".

Reply:
I think Topher made some PR to try to make all this more consistent but his had issues as well. I'm up for changes. This is some of the silliness that's been there since MOOSE was a wee-little project!